tests/functional/job-submission/08-activity-log-host.t

```diff
@@ -18,23 +18,17 @@
 # Test job submission, activity log has remote host name
 export CYLC_TEST_IS_GENERIC=false
 . "$(dirname "$0")/test_header"
-set_test_remote_host
-set_test_number 4
+require_remote_platform
+set_test_number 2
 
 install_suite "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
 
 run_ok "${TEST_NAME_BASE}-validate" \
-    cylc validate "${SUITE_NAME}" -s "CYLC_TEST_HOST=${CYLC_TEST_HOST}"
+    cylc validate "${SUITE_NAME}" -s "CYLC_REMOTE_PLATFORM=${CYLC_REMOTE_PLATFORM}"
 suite_run_ok "${TEST_NAME_BASE}-run" \
-    cylc run --debug --no-detach --reference-test -s "CYLC_TEST_HOST=${CYLC_TEST_HOST}" \
+    cylc run --debug --no-detach --reference-test -s "CYLC_REMOTE_PLATFORM=${CYLC_REMOTE_PLATFORM}" \
     "${SUITE_NAME}"
 
-RUN_DIR="$(cylc get-global-config --print-run-dir)/${SUITE_NAME}"
-grep_ok "^(${CYLC_TEST_HOST}) .*\\[STDOUT\\]" \
-    "${RUN_DIR}/log/job/19990101T0000Z/sleeper/01/job-activity.log"
-grep_ok "^(${CYLC_TEST_HOST}) .*\\[STDOUT\\]" \
-    "${RUN_DIR}/log/job/19990101T0000Z/sleeper/02/job-activity.log"
-
-purge_suite_remote "${CYLC_TEST_HOST}" "${SUITE_NAME}"
+purge_suite_remote "${CYLC_REMOTE_PLATFORM}" "${SUITE_NAME}"
```

Comment:
Do we not need to check here that the activity log has a remote name? If not should we change the description of the test?

Reply:
I added this test.

Reply:
I think that I have become a bit confused. I've asked Oliver to check that this hasn't been broken by the remrun stuff he did, because this is looking for a log message in the job-activity.log which I'm not sure that `batch_system_manager` can write anymore. In which case, I think that this test is defunct.

Reply:
See my review comment on `job-submission/09-blah/suite.rc` - this is similar, we don't log a `(HOSTNAME)` prefix on these lines, on this branch.  Should probably grep the host name in  the jobs-submit cmd ERROR line in the suite log instead.

Reply:
I added a test grepping the hostname in the retrieved `job.out` file, where the job script was `echo $HOSTNAME`. Sufficient? (It was my belief in the sufficiency of this that caused me to write "I added this test")